### PR TITLE
zeusgang.1bt.uk

### PR DESCRIPTION
**Pull request by zeusgangproject**
### Link to Website

Link:

### CODE REVIEW

File: domains/zeusgang.1bt.uk.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+  "description": "Browser extension Random User-Agent homepage",
```
Comment (tarampampam) [inline domains/zeusgang.1bt.uk.json:3]: Is that the correct description?

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: domains/zeusgang.1bt.uk.json (hunk 1 above)
Comment (zeusgangproject) [reply domains/zeusgang.1bt.uk.json:3]: nah I'm sorry, I actually copied someone else's, but for the website it's just the team's personal website

Comment (zeusgangproject) [reply domains/zeusgang.1bt.uk.json:3]: and i already starred this repo

Comment (tarampampam) [reply domains/zeusgang.1bt.uk.json:3]: change this line to something relevant, please

**Pull request closed without merging**